content/git/concepts/reset/reset.md

```diff
@@ -0,0 +1,65 @@
+---
+Title: 'Reset'
+Description: 'In Git, reset is used to set the HEAD pointer back to a previous commit and optionally undo staged changes and the working tree.'
+Subjects:
+  - 'Bash/Shell'
+  - 'Developer Tools'
+Tags:
+  - 'Git'
+  - 'Version Control'
+CatalogContent:
+  - 'learn-the-command-line'
+  - 'learn-git'
+---
+
+In Git, the `reset` command is used to change the state of the Git repository or undo commits.
+
+## Syntax
+
+In the terminal, the "git reset" command can be used by typing the following:
+
+```pseudo
+git reset <mode-option> <commit-reference-number>
+```
+
+### Mode Options
+
+The `<mode-options>` refers to how far `reset` will go when rolling back changes to a previous commit, including:
+
+- Where the `HEAD` is pointing towards (usually done with just `git reset`).
+- Whether the staging area or Index, reflects the commit the `HEAD` is now pointing towards.
+- Whether the working tree is rolled back to reflect the changes reset in the `HEAD` and staging area.
+
+More specifically, these options include:
+
+1. --soft: This undoes the commit but still shows the file as staged and permits only new commits.
+2. --mixed: This undoes the commit and unstages the file and the file can be edited and committed normally.
+3. --hard: This removes the new commit including all changes and uncommitted files.
+
+The commit can either be the commit hash generated by Git, or in reference to the HEAD.
+
+## Example
+
+Suppose a file named **example.txt** is accidentally committed to a Git repository. To undo the commit, the command `git reset --mixed HEAD~1` is used in the terminal. `--mixed` is the default mode, so `git reset HEAD~1` will achieve the same result. This resets the HEAD and index to the previous commit.
```

Comment:
```suggestion
If an error was made in a file named **example.txt** and accidentally added and committed to a Git repository, `git reset` can be used to go back to before that happened. Using the default `--mixed` mode resets the `HEAD` and staging area back to the previous commit.
```

Reply:
Hey, so I sort of changed it a bit and was wondering if that is fine?